wagtailio/static/sass/components/_get-started.scss

```diff
@@ -0,0 +1,53 @@
+.get-started {
+    padding-bottom: 50px;
+
+    @include media-query(medium) {
+        padding-bottom: 100px;
+    }
+
+    @include media-query(large) {
+        padding-bottom: 200px;
+    }
+
+    &__heading {
+        grid-column: 2 / span 2;
+
+        @include media-query(medium) {
+            grid-column: 2 / span 3;
+            margin-bottom: 40px;
+        }
+
+        @include media-query(large) {
+            grid-column: 2 / span 5;
+            margin-bottom: 100px;
+        }
+    }
+
+    &__list {
+        display: flex;
+        flex-direction: column;
+        list-style: none;
+        padding: 0;
+        margin: 0;
+        row-gap: 20px;
+        grid-column: 2 / span 2;
+
+        @include media-query(medium) {
+            grid-column: 2 / span 3;
+            row-gap: 50px;
+        }
+
+        @include media-query(large) {
+            grid-column: 2 / span 5;
+            flex-flow: row wrap;
+            row-gap: 100px;
+        }
+    }
+
```

Comment:
Note: The icons are a little large. Not massively off, it would be good to hear if you're aiming for consistency across all of the other components.
![image](https://user-images.githubusercontent.com/22497837/182811621-ba6137e1-dc30-4669-ac6e-e944db3cf02d.png)

Reply:
Definitely want to match the design so i've reduced these down. Thanks.